docs/architecture.md

```diff
@@ -1,3 +1,16 @@
 # Architecture
 
-To be added
\ No newline at end of file
+To clarify standpoint of Pouch in container system, we construct it with explicit architecture. To ensure the clear separation of functionality, we have organized Pouch with components. Therefore, when mentioning architecture, we usually include two parts:
+
+* ecosystem architecture
+* component architecture
+
+## Ecosystem Architecture
+
+In pouch's roadmap, we set ecosystem embracing as a big target. To upper orchestrating layer, pouch supports Kubernetes and Swarm. To underlying runtime layer, pouch is compatible with oci-compatible runtime, such as runC, runV, runlxc and so on. To make storage and network big supplements, CNI and CSI are in scope right there.
```

Comment:
Shall we add the link of abbreviation(CNI, CSI and runC)?

Reply:
Good point. Accepted and updated @CodeJuan